types/office-js/index.d.ts

```diff
@@ -8271,7 +8287,7 @@ declare namespace Office {
          *
          * When including links in HTML markup, you can disable online link preview by setting the id attribute on the anchor (<a>) to "LPNoLP" (please see the Examples section for a sample).
          *
-         * [Api set: Mailbox 1.1]
+         * [Api set: Mailbox 1.2]
```

Comment:
Can you make sure to change this in the legacy docs afterwards? I can show you how to do that without rerunning everything.

Reply:
For sure.